src/foam/u2/layout/Layouts.js

```diff
@@ -1,23 +1,207 @@
 foam.CLASS({
   package: 'foam.u2.layout',
-  name: 'RowLayout',
+  name: 'Rows',
   extends: 'foam.u2.Element',
+
+  requires: [
+    'foam.u2.layout.Row',
+  ],
+
+  documentation: `
+    A layout for a group of rows layed out vertically.
+  `,
+
   methods: [
     function add() {
-      console.log('Adding row!');
-      return this.SUPER.apply(this, arguments);
+      /**
+       * ! Trying to add a border like with Col breaks the row, we can look into later
+       * ! As card rows aren't probable right now
+       * Here we are checking for two cases
```

Comment:
This kind of comment isn't super useful since it's basically 1:1 with the actual code so it's kind of redundant and runs the risk of going stale as soon as the code is modified. Comments should typically explain *why* the code is doing what it's doing rather than just reiterating what the code is doing. If you want to add some documentation you can say something like:

> This class expects all child elements to be instances of foam.u2.layout.Row so we override the add method to enforce this.


This kind of comment is a little more high level so doesn't risk going stale as easily.

Reply:
Done.